students/stuartkershaw/lesson03/mailroom.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+donors = [
+    'Anne Hathaway', 'Geronimo Jackson', 'Dingo Dogson',
+    'Leo Kottke', 'Flask McCreole', 'Piglet Norquist',
+    [[4000, 6250], [150, 400, 1250], [150, 300, 100],
+     [75, 125], [1000], [20, 15, 30]]
+]
+
```

Comment:
I would suggest combining each donor and donation into a single list here for easier lookups and additions to the overall data structure. This should allow you to simplify some of the code in your 'set_donation`, `get_donor_name` and `show_donor_table` functions. 

Something like this (kind of like rows in a database):
```
donors = [
    ['Anne Hathaway', [4000, 6250]],
    ['Geronimo Jackson', [4000, 6250]],
]
```